src/domain/proof-of-liabilities/generate-leaves.ts

```diff
@@ -0,0 +1,19 @@
+import { createHash } from "crypto"
+
+export const getHash = (data: string): string => {
+  const hash = createHash("sha256")
+  return hash.update(data).digest("hex")
+}
+
+export const GenerateLeaves = (liabilities: Liabilities[]): TreeNode[] => {
+  const leaves: TreeNode[] = []
+  liabilities.forEach((liability: Liabilities, idx: number) => {
+    const hash = getHash(`${liability.accountId}${liability.walletBalance}${idx}`)
```

Comment:
The logic for constructing the hash from a liability should be isolated in a function.